cinder/volume/manager.py

```diff
@@ -5042,11 +5152,19 @@ def _connection_terminate(self,
             # so if we don't have a connector we can't terminate a connection
             # that was never actually made to the storage backend, so just
             # log a message and exit.
-            LOG.debug('No connector for attachment %s; skipping storage '
-                      'backend terminate_connection call.', attachment.id)
+            msg = ('No connector for attachment %s; skipping storage '
+                   'backend terminate_connection call.' % attachment.id)
             # None indicates we don't know and don't care.
```

Comment:
This comment should be moved down to the return.

Reply:
done